src/main/kotlin/ro/hibyte/betting/service/BetTemplateService.kt

```diff
@@ -0,0 +1,37 @@
+package ro.hibyte.betting.service
+
+import org.springframework.stereotype.Service
+import ro.hibyte.betting.entity.BetTemplate
+import ro.hibyte.betting.exceptions.types.BetTemplateNotFoundException
+import ro.hibyte.betting.exceptions.types.EntityAlreadyExistsException
+import ro.hibyte.betting.repository.BetTemplateRepository
+
+@Service
+class BetTemplateService(private val betTemplateRepository: BetTemplateRepository) {
+
+    fun createBetTemplate(betTemplate: BetTemplate): BetTemplate {
```

Comment:
Same naming thing as for the controllers

Reply:
Done.